macros/edr/data_monitoring/monitors_query/get_start_bucket_in_data.sql

```diff
@@ -1,6 +1,6 @@
 {% macro get_start_bucket_in_data(timestamp_column, min_bucket_start, time_bucket) %}
     {% set bucket_start_datediff_expr %}
-      floor({{ elementary.timediff(time_bucket.period, min_bucket_start, elementary.edr_cast_as_timestamp(timestamp_column)) }} / {{ time_bucket.count }}) * {{ time_bucket.count }}
+      floor({{ elementary.edr_datediff(min_bucket_start, elementary.edr_cast_as_timestamp(timestamp_column), time_bucket.period) }} / {{ time_bucket.count }}) * {{ time_bucket.count }}
```

Comment:
I have to say moving from `timediff` to `datediff` looked to me like it would truncate to dates but it doesn't, after checking in dbt-core

Reply:
Is this an actionable comment?

Reply:
No - more like something for future readers wondering if it's truncating or not - it doesn't.